ush/load_ufsda_modules.sh

```diff
@@ -34,13 +34,13 @@ source "${HOMEgfs}/ush/module-setup.sh"
 module use "${HOMEgfs}/sorc/gdas.cd/modulefiles"
 
 case "${MACHINE_ID}" in
-  ("hera" | "orion" | "hercules")
+  ("hera" | "orion" | "hercules" | "wcoss2")
```

Comment:
```suggestion
  ("hera" | "orion" | "hercules" | "wcoss2" | "acorn")
```

Reply:
Good catch @CatherineThomas-NOAA .   Thank you @WalterKolczynski-NOAA for the suggestion.  `feature/wcoss2_ufsda` updated at 7ebfdde3